Source/OxyPlot/Rendering/OxyAnnulus.cs

```diff
@@ -0,0 +1,229 @@
+            }
+        }
+
+        /// <summary>
+        /// Gets the inner top of the hypoid.
+        /// </summary>
+        /// <value>The top inner point.</value>
+        public ScreenPoint InnerTop => new ScreenPoint(this.Center.X, this.Center.Y + this.InnerRadius);
+
+        /// <summary>
+        /// Gets the inner bottom of the hypoid.
+        /// </summary>
+        /// <value>The bottom inner point.</value>
+        public ScreenPoint InnerBottom => new ScreenPoint(this.Center.X, this.Center.Y - this.InnerRadius);
+
+        /// <summary>
+        /// Gets the inner left of the hypoid.
+        /// </summary>
+        /// <value>The most left inner point.</value>
+        public ScreenPoint InnerLeft => new ScreenPoint(this.Center.X - this.InnerRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the inner right of the hypoid.
+        /// </summary>
+        /// <value>The most right inner point.</value>
+        public ScreenPoint InnerRight => new ScreenPoint(this.Center.X + this.InnerRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the outer top of the hypoid.
+        /// </summary>
+        /// <value>The top outer point.</value>
+        public ScreenPoint OuterTop => new ScreenPoint(this.Center.X, this.Center.Y + this.OuterRadius);
+
+        /// <summary>
+        /// Gets the outer bottom of the hypoid.
+        /// </summary>
+        /// <value>The bottom outer point.</value>
+        public ScreenPoint OuterBottom => new ScreenPoint(this.Center.X, this.Center.Y - this.OuterRadius);
+
+        /// <summary>
+        /// Gets the outer left of the hypoid.
+        /// </summary>
+        /// <value>The most left outer point.</value>
+        public ScreenPoint OuterLeft => new ScreenPoint(this.Center.X - this.OuterRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the outer right of the hypoid.
+        /// </summary>
+        /// <value>The most right outer point.</value>
+        public ScreenPoint OuterRight => new ScreenPoint(this.Center.X + this.OuterRadius, this.Center.Y);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OxyAnnulus"/> structure that has a specified center a inner and outer radius
+        /// </summary>
+        /// <param name="center">The center location of the hypoid</param>
+        /// <param name="r1">The inner radius of the hypoid</param>
+        /// <param name="r2">The outer radius of the hypoid</param>
+        /// <exception cref="System.ArgumentOutOfRangeException">r1;The inner radius should not be negative.</exception>
+        /// <exception cref="System.ArgumentOutOfRangeException">r2;The outer radius should not be equal or smaller as the inner radius.</exception>
+        public OxyAnnulus(ScreenPoint center, double r1, double r2)
+        {
+            if (r1 < 0)
+                throw new ArgumentOutOfRangeException("r1", "The inner radius should not be negative.");
```

Comment:
Should use `nameof(r1)` etc. to minimise refactoring burden

Reply:
Would prefer `innerRadius`, `outerRadius` as the parameter names themselves

Reply:
agreed & changed